packages/atomic/src/locales.json

```diff
@@ -15,6 +19,12 @@
     "en": "Clear",
     "fr": "Effacer"
   },
+  "showMore": {
+    "en": "show more"
+  },
+  "showLess": {
+    "en": "show less"
```

Comment:
In french: `Voir moins de résultats`.

You can also copy paste them from `strings.json` in JSUI repo.

Reply:
Yes there is possibly more languages for these strings already